source/application/AMPIC887DriverServer/AMGCS2.h

```diff
@@ -0,0 +1,107 @@
+#ifndef AMGCS2_H
+#define AMGCS2_H
+
+#include <QChar>
+/*!
+  * Namespace which contains all the enumerators for use with the GCS2.0 syntax
+  * provided by the PI C887.11 controllers.
+  */
+namespace AMGCS2 {
+
+	/***************
+	  * Enums
+	  ****************************/
+
+	/*!
+	  * Enumerates all the hexapod axes controlled by a C887.11 controller.
+	  */
+	enum Axis {
+		UnknownAxis = -1,
+		XAxis = 0,
```

Comment:
I've always struggled with this too. I think that Iain might be right here.
If you want to use this type, then it's AMGCS2::Axis
But if you want to set or compare a value, right now, its AMGCS2::XAxis which seems better than AMGCS2::X
I've taken to being really pedantic with the enum names to avoid the confusion.